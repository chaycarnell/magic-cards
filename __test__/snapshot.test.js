import React from "react";
import renderer from "react-test-renderer";
import Card from "../src/components/card";
import CardTable from "../src/components/card_table";

it("renders one card correctly", () => {
  const card = { suit: "Clubs", title: "2" };
  const tree = renderer.create(<Card card={card} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the card table correctly with 1 card", () => {
  const cards = [{ suit: "Clubs", title: "2", rank: 0 }];
  const tree = renderer.create(<CardTable drawnCards={cards} />).toJSON();
  expect(tree).toMatchSnapshot();
});
