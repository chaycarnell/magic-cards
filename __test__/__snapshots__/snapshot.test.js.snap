// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders one card correctly 1`] = `
<div
  className="sc-bdVaJa hVKLoi"
>
  <div
    className="sc-bwzfXH kcAiZW"
  >
    2
  </div>
  <div
    className="sc-htpNat fvkhKw"
  >
    <div
      className="sc-bxivhb hhpgyT"
      type="Clubs"
    />
  </div>
</div>
`;

exports[`renders the card table correctly with 1 card 1`] = `
<div
  className="sc-ifAKCX eFpWkd"
>
  <h2
    className="sc-bZQynM GbkAN"
  >
    Card Table
  </h2>
  <div
    className="sc-bdVaJa hVKLoi"
  >
    <div
      className="sc-bwzfXH kcAiZW"
    >
      2
    </div>
    <div
      className="sc-htpNat fvkhKw"
    >
      <div
        className="sc-bxivhb hhpgyT"
        type="Clubs"
      />
    </div>
  </div>
</div>
`;
